### DISCUSSION TITLE
I can't clone the repository, even though my internet connection is fine.

### DISCUSSION THREAD

**Discussion by Laggar93** [community/community#108464] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello, I can't clone the repository, even though my internet connection is fine.

error: RPC failed; curl 18 transfer closed with outstanding read data remaining
error: 7146 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: index-pack failed

git config --global http.postBuffer 524288000 not help to me

how I can fix it? thanks!

**Comment by patrickwilliamwalker** [comment]
Welcome to 'software engineering.'
